### ISSUE TITLE
[Bug] lua error

### ISSUE THREAD

**Issue by Nihlo** [Alumian/CEPGP-Retail#27]
**The Error**

Message: Interface\AddOns\CEPGP\Loot.lua:49: 'for' limit must be a number
Time: Thu Sep 19 19:33:57 2019
Count: 2
Stack: Interface\AddOns\CEPGP\Loot.lua:49: 'for' limit must be a number
Interface\AddOns\CEPGP\Loot.lua:49: in function `CEPGP_LootFrame_Update'
Interface\AddOns\CEPGP\Handlers.lua:420: in function `CEPGP_handleLoot'
Interface\AddOns\CEPGP\Core.lua:142: in function `CEPGP_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: items = <table> {
}
count = 0
numLootItems = nil
texture = nil
item = nil
quantity = nil
quality = nil
(for index) = 1
(for limit) = nil
(for step) = 1
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "'for' limit must be a number"

**Describe the bug**
lua error on looting in raid

**To Reproduce**
How can this bug be reproduced?

**Debug Information**
Type /cepgp debug and copy and paste the information into this bug report.

**Comment by Alumian** [comment]
Hi Nihlo,

Please follow the instructions in the bug report template. Please type /cepgp debug and copy and paste the information into the report. Also, are you using ElvUI?

**Comment by NooBiToo** [comment]
> Hi Nihlo,
> 
> Please follow the instructions in the bug report template. Please type /cepgp debug and copy and paste the information into the report. Also, are you using ElvUI?

I too use ElvUI

**Comment by Alumian** [comment]
> > Hi Nihlo,
> > Please follow the instructions in the bug report template. Please type /cepgp debug and copy and paste the information into the report. Also, are you using ElvUI?
> 
> I too use ElvUI

Are you having this same issue?

**Comment by NooBiToo** [comment]
> > > Hi Nihlo,
> > > Please follow the instructions in the bug report template. Please type /cepgp debug and copy and paste the information into the report. Also, are you using ElvUI?
> > 
> > 
> > I too use ElvUI
> 
> Are you having this same issue?

yep

**Comment by Alumian** [comment]
Okay that makes sense. I did stumble across this issue myself since I also use ElvUI. I believe it's to do with ElvUI overwriting the stock blizzard lootframe update function. I've just finished another bugfix this morning so when I get home from work I'll work on implementing a solution for this.
I'll try to have the solution pushed today but it'll depend on my initial investigation.

**Comment by NooBiToo** [comment]
> Okay that makes sense. I did stumble across this issue myself since I also use ElvUI. I believe it's to do with ElvUI overwriting the stock blizzard lootframe update function. I've just finished another bugfix this morning so when I get home from work I'll work on implementing a solution for this.
> I'll try to have the solution pushed today but it'll depend on my initial investigation.

Yep, I checked, and if I turn off the improvements of the elvui, then the error is not caused

**Comment by Alumian** [comment]
Ok thanks NooBiToo, I'll look into that when I get home.

**Comment by Nihlo** [comment]
I can check about the debug message tonight, not using ElvUI though. Not any custom lootunitframe or complete UI.

**Comment by Alumian** [comment]
Hi Nihlo,
I just pushed an update which should have fixed this issue.
Please let me know if you're still encountering the issue.

**Comment by Nihlo** [comment]
Hadn't had any errors while raiding but got a new one doing 5mans after that:

`Message: Interface\AddOns\CEPGP\Loot.lua:6: 'for' limit must be a number
Time: Tue Sep 24 23:50:25 2019
Count: 1
Stack: Interface\AddOns\CEPGP\Loot.lua:6: 'for' limit must be a number
Interface\AddOns\CEPGP\Loot.lua:6: in function `CEPGP_LootFrame_Update'
Interface\AddOns\CEPGP\Handlers.lua:423: in function `CEPGP_handleLoot'
Interface\AddOns\CEPGP\Core.lua:139: in function `CEPGP_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: items = <table> {
}
count = 0
numLootItems = nil
texture = nil
item = nil
quantity = nil
quality = nil
(for index) = 1
(for limit) = nil
(for step) = 1
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "'for' limit must be a number"
`

**Comment by mykkenny** [comment]
Is there a workaround of some sort, something to disable in ElvUI in the short term to fix this?
